### PR TITLE
Refactor: more defloating of fixedpoint initialisations

### PR DESCRIPTION
**Pull request by vhavrylov-ncube**
+ replaced fload/double initialisation with string constructors

### CODE REVIEW

File: libs/math/tests/unit/fixed_point/fixed_point.cpp [hunk 1]
```diff
@@ -339,7 +339,7 @@ TEST(FixedPointTest, FromString_64_64)
   EXPECT_EQ(m_one_point_five, m_one_point_five_flt);
 
   fp128_t e1("2.718281828459045235360287471352662498");
-  fp128_t e2(2.718281828459045235360287471352662498);
+  fp128_t e2("2.718281828459045235360287471352662498");
   EXPECT_TRUE(e1.Near(e2));
   EXPECT_NE(e1, e2);
```
Comment (evsmithx) [inline libs/math/tests/unit/fixed_point/fixed_point.cpp]: I would expect e1 == e2 now? I'm not sure this test tests anything meaningful any more!

File: libs/ml/tests/unit/layers/scaled_dot_product_attention.cpp [hunk 2]
```diff
@@ -19,6 +19,7 @@
 #include "ml/layers/scaled_dot_product_attention.hpp"
 
 #include "gtest/gtest.h"
+#include "ml/layers/scaled_dot_product_attention.hpp"
```
Comment (evsmithx) [inline libs/ml/tests/unit/layers/scaled_dot_product_attention.cpp:20]: duplicate include

**Review by evsmithx — CHANGES_REQUESTED** [review]
Can you get @markos to check the fixed_point.cpp test to make sure they're still doing the right thing?

File: libs/math/tests/unit/fixed_point/fixed_point.cpp (hunk 1 above)
Comment (vhavrylov-ncube) [reply libs/math/tests/unit/fixed_point/fixed_point.cpp]: I guess you're right, it's a mistake; I wouldn't touch this test then, let @markos review it later; it is not critical change anyway.

File: libs/ml/tests/unit/layers/scaled_dot_product_attention.cpp (hunk 2 above)
Comment (vhavrylov-ncube) [reply libs/ml/tests/unit/layers/scaled_dot_product_attention.cpp:20]: removed

**Review by markos — APPROVED** [review]
Looks Good(TM)

**Review by evsmithx — APPROVED** [review]

**Pull request merged**